meetings/2023-04-12.md

```diff
@@ -0,0 +1,50 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2023-04-12
+
+## Links
+
+* **Recording**:  <https://www.youtube.com/watch?v=eYWLgLN3n3s>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1369>
+
+## Present
+
+* Beth Griggs @BethGriggs (voting member)
+* Colin Ihrig @cjihrig (voting member)
+* Danielle Adams @danielleadams (voting member)
+* Chengzhong Wu @legendecas (voting member)
+* Michael Dawson @mhdawson (voting member)
+* Rafael Gonzaga @RafaelGSS (voting member)
+* Darshan Sen @RaisinTen (voting member)
+* Richard Lau @richardlau (voting member)
+
+## Agenda
+
+### Announcements
+
+* Danielle, new version of 18 this week
+* Rafael, new version of 19 this week
+* Rafael - Node.js 20 Next Week
+* Michael - Open JS and collab summit
+* Michael - new meeting times, heads up for anybody who watches online, check out the calendar for the updates.
+
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* Rich through comment on meeting issue - There are plans for the CPC and a third board seat. See openjs-foundation/cross-project-council#1054
+
+### nodejs/node
+
+* Improve webstreams brand validations [#47218](https://github.com/nodejs/node/pull/47218)
+  * Looks like the conversation was resolved.
+
+## Strategic Initiatives
+
+* Chengzhong Wu - Shadow Realm, Ongoing discussion  related to memory leak, if goes well will submit PR . Discussion in issue <https://github.com/nodejs/node/issues/47353>
+* Darshan - single executable applications, just landed in 18.x.  PR that creates layout, allowing multiple resources (thanks to Joyee), this will be breaking
```

Comment:
```suggestion
* Darshan Sen - single executable applications, just landed in 18.x.  PR that creates layout, allowing multiple resources (thanks to Joyee), this will be breaking
```